### PR TITLE
Add unit test

### PR DESCRIPTION
**Pull request by nalakas**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: component/src/test/java/org/wso2/carbon/identity/authenticator/bitly/BitltTestClass2.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
+package org.wso2.carbon.identity.authenticator.bitly;
```
Comment (kesavany) [inline component/src/test/java/org/wso2/carbon/identity/authenticator/bitly/BitltTestClass2.java]: Add License header.

File: component/src/test/resources/testng.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+<suite name="Suite1" verbose="1" >
+
+    <test name="bitly-Authenticator-Unit-Tests">
+        <classes>
+            <class name="org.wso2.carbon.identity.authenticator.bitly.BitlyTest"/>
+            <class name="org.wso2.carbon.identity.authenticator.bitly.BitltTestClass2"/>
+        </classes>
+    </test>
+</suite>
```
Comment (shilmyhasan) [inline component/src/test/resources/testng.xml]: add new line EOF

File: component/src/test/java/org/wso2/carbon/identity/authenticator/bitly/BitlyTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                bitlyAuthenticator.getConfigurationProperties().get(2).getName());
+        Assert.assertEquals(3, bitlyAuthenticator.getConfigurationProperties().size());
+    }
+
+    @Test(description = "Test case for getAccessRequest", dataProvider = "authenticatorProperties")
+    public void testGetAccessRequest(Map<String, String> authenticatorProperties) throws Exception {
+        OAuthClientRequest accessRequest = Whitebox.invokeMethod(bitlyAuthenticator, "getAccessRequest",
+                authenticatorProperties.get("tokenEndpoint"), authenticatorProperties.get(OIDCAuthenticatorConstants.CLIENT_ID),
+                authenticatorProperties.get("code"), authenticatorProperties.get(OIDCAuthenticatorConstants.CLIENT_SECRET),
+                authenticatorProperties.get("callbackUrl"));
```
Comment (Rajith90) [inline component/src/test/java/org/wso2/carbon/identity/authenticator/bitly/BitlyTest.java]: Use constants if possible for "tokenEndpoint", "code" and etc even though this  is a test case

File: component/src/test/java/org/wso2/carbon/identity/authenticator/bitly/BitltTestClass2.java (hunk 1 above)
Comment (nalakas) [reply component/src/test/java/org/wso2/carbon/identity/authenticator/bitly/BitltTestClass2.java]: fixed in 9244d37

File: component/src/test/resources/testng.xml (hunk 2 above)
Comment (nalakas) [reply component/src/test/resources/testng.xml]: fixed in 9244d37

File: component/src/test/java/org/wso2/carbon/identity/authenticator/bitly/BitlyTest.java (hunk 3 above)
Comment (nalakas) [reply component/src/test/java/org/wso2/carbon/identity/authenticator/bitly/BitlyTest.java]: add constants in 2085640

**Pull request merged**